packages/plugins/Scale/src/components/Scale.vue

```diff
@@ -15,12 +32,57 @@
 
 <script lang="ts">
 import Vue from 'vue'
-import { mapGetters } from 'vuex'
+import { mapGetters, mapMutations, mapActions } from 'vuex'
+import thousandsSeparator from '../utils/thousandsSeperator'
+import beautifyScale from '../utils/beautifyScale'
 
 export default Vue.extend({
   name: 'PolarScale',
   computed: {
-    ...mapGetters('plugin/scale', ['scaleToOne', 'scaleWithUnit']),
+    ...mapGetters('plugin/scale', [
+      'scaleToOne',
+      'scaleValue',
+      'scaleWithUnit',
+      'showScaleSwitcher',
+      'zoomOptions',
+    ]),
+    scaleNumberToScale() {
+      return (scale: number) => {
+        return '1 : ' + thousandsSeparator(scale)
+      }
+    },
+    scale: {
+      get() {
+        const scaleToCompare = beautifyScale(this.scaleValue)
+        const bestMatchingScale = this.zoomOptions.reduce((prev, curr) => {
+          return Math.abs(curr.scale - scaleToCompare) <
+            Math.abs(prev.scale - scaleToCompare)
+            ? curr
+            : prev
+        })
+        return bestMatchingScale.scale
+      },
+      set(value: number) {
+        this.setScaleValue(value)
+      },
+    },
+    showSelectOptions() {
+      return this.showScaleSwitcher
+    },
```

Comment:
I just had the idea that the `select` of the scale switcher does not get used if `@polar/plugin-zoom` is not available and the normal scale is displayed instead,
One could add a new parameter to `data` called `zoomPluginAvailable` which is set to `false` in `mounted` in that case and `showSelectOptions` is also reliant on that parameter.

Feel free to add it or leave it as is right now 🎩

Reply:
74cf864f134df9513118b6b33e0c1223a18f776a